_posts/2023-12-13-basicswapdex-firo-installer.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Firo is Now Integrated into BasicSwap Installer (.exe)"
+summary: "Compatible with Bidirectional Adaptor Signature Atomic Swaps"
+tags: [news, dev]
+author: "Augustus Jong"
+img: "/blog/assets/basicswapdexblog/banner.jpg"
+---
+
+We are thrilled to announce that the [BasicSwap](https://basicswapdex.com/) installation wizard now comes with complete Firo compatibility! This means you can easily install Firo, featuring our recent groundbreaking [Lelantus Spark update](https://firo.org/2023/11/27/lelantus-spark-binaries-release.html), directly from the automated installer (remember to run it as an administrator).
+
+## About Basicswap
+
+BasicSwap stands as a decentralized, cross-chain trading exchange (DEX) that ensures privacy and security without relying on a central point of failure. This platform empowers users to seamlessly exchange cryptocurrencies, such as Bitcoin or Firo, free from restrictions or third-party intervention. Crafted by the cypherpunks at Particl, BasicSwap is deliberately constructed with an unwavering commitment to uncompromised privacy and security. This design is a direct response to the emerging challenges to freedom, reflecting a proactive stance against potential threats.
+
+## Bidirectional Adaptor Signature Atomic Swaps?
+
+Firo can swap as both the offerer and the participant of an adaptor signature swap.  Adaptor signature swaps present themselves as multi-signature transactions, concealing the true nature of the transaction and leaving no trace suggesting a connection between two transactions on distinct blockchains
+
+To learn more head to [Particl's blog](https://particl.news/firo-now-compatible-with-bidirectional-adaptor-signature-atomic-swaps/) for a comprehensive explanation.
+
+### How to Download
+
+![](/blog/assets/basicswapdexblog/installer.png)

```

Comment:
```suggestion
![Screenshot of basicswap's installation wizard listing Firo](/blog/assets/basicswapdexblog/installer.png)

```

Alt text is very useful for visually impaired users and increases the SEO score.